File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/Services/WindowServiceViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.POCO;
using System.Windows;

namespace MVVMDemo.Services {
    public class WindowServiceViewModel {
        public void ShowRegistrationForm() {
            WindowServiceDetailViewModel detailViewModel = WindowServiceDetailViewModel.Create();

            IWindowService service = this.GetRequiredService<IWindowService>();
            service.Title = "Registration Form";
            service.Show(viewModel: detailViewModel);
        }
    }
}

File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/Services/WindowServiceViewModel.vb
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.POCO

Namespace MVVMDemo.Services

    Public Class WindowServiceViewModel

        Public Sub ShowRegistrationForm()
            Dim detailViewModel As WindowServiceDetailViewModel = WindowServiceDetailViewModel.Create()
            Dim service As IWindowService = GetRequiredService(Of IWindowService)()
            service.Title = "Registration Form"
            service.Show(viewModel:=detailViewModel)
        End Sub
    End Class
End Namespace
